tests/gameResource.spec.js

```diff
@@ -67,13 +67,13 @@ describe("resource : gameResource", function() {
   });
 
   beforeEach(function() {
-    spyOn(yf, "api").andCallThrough();
+    spyOn(yf, "api").and.callThrough();
   });
 
   // leagues
   describe(": leagues", function() {
     it("should build a proper url to retrieve league data for a single league using a numeric game key", function() {
-      game.leagues(328, "328.l.34014", null);
+      game.leagues(328, "328.l.34014", () => {});
```

Comment:
Just need to provide some callback, even though we aren't testing the value called-back-with.